src/js/app.js

```diff
@@ -91,6 +91,49 @@ $(document).ready(function() {
     }
   });
 
+  //Add toggle effect for sliding #menu
+  let showingMenu = false;
+  const $menu = $('#menu');
+  const $main = $('main');
+  const config2 = {
+    duration: 1000,
+    easing: 'easeOut'
+  };
+  $('#menu-burger,.close-menu,#navigation-links>li>a').on('click', function() {
+    if (showingMenu) {
+      $menu.removeClass('active');
+      $main.removeClass('menu-active');
+      showingMenu = false;
+      event.preventDefault();
+    } else {
+      $menu.addClass('active');
+      $main.addClass('menu-active');
+      showingMenu = true;
+      event.preventDefault();
+    }
+    return showingMenu;
```

Comment:
I actually don't understand when to use return, so I often put it in there when things don't work and hope it's the answer....